htdocs/api/v0.0.2/Candidates.php

```diff
@@ -121,17 +121,29 @@ public function handlePOST()
                     $data['Candidate']['Gender'],
                     $data['Candidate']['PSCID']
                 );
+
+                if (isset($data['Candidate']['Project'])) {
+                    $projectName = htmlspecialchars($data['Candidate']['Project']);
```

Comment:
Why is this being HTML escaped? This is dealing with json, not HTML. a "<" or "&" is perfectly valid within the context of JSON.